src/templates/index.html

```diff
@@ -14,12 +14,21 @@ <h1 class="display-3">Word Finder</h1>
                         <select class="custom-select my-2" name="sellanguage">
                             <option selected="">Select a language</option>
                             <option value="1">Chinese</option>
-                            <option value="2">English</option>
-                            <option value="3">French</option>
-                            <option value="4">Italian</option>
-                            <option value="5">Japanese</option>
-                            <option value="6">Korean</option>
-                            <option value="7">Russian</option>
+                            <option value="2">Dutch</option>
+                            <option value="3">English</option>
+                            <option value="4">Finnish</option>
+                            <option value="5">French</option>
+                            <option value="6">German</option>
+                            <option value="7">Greek</option>
+                            <option value="8">Hungarian</option>
+                            <option value="9">Italian</option>
+                            <option value="10">Japanese</option>
+                            <option value="11">Korean</option>
+                            <option value="12">Latin</option>
+                            <option value="13">Polish</option>
+                            <option value="14">Portuguese</option>
+                            <option value="15">Russian</option>
+                            <option value="16">Spanish</option>
                         </select>
```

Comment:
Adding New Languages is done, Nice work.

Reply:
Is there a way to link the option values (html) to the language_list var in util.py?